sensing/autoware_pointcloud_preprocessor/src/outlier_filter/polar_voxel_outlier_filter_node.cpp

```diff
@@ -993,6 +1102,107 @@ PolarVoxelOutlierFilterComponent::extract_polar_from_xyz(float x, float y, float
   return polar;
 }
 
+void PolarVoxelOutlierFilterComponent::publish_area_marker(
+  const std_msgs::msg::Header & input_header)
+{
+  auto marker = visualization_msgs::msg::Marker();
+  marker.header = input_header;
+  marker.ns = "visibility_estimation_area";
+  marker.id = 0;
+  marker.type = visualization_msgs::msg::Marker::TRIANGLE_LIST;
+  marker.action = visualization_msgs::msg::Marker::ADD;
+  marker.scale.x = 1.0;
+  marker.scale.y = 1.0;
+  marker.scale.z = 1.0;
+  marker.color.a = 0.1;
+  marker.color.r = 0.0;
+  marker.color.g = 1.0;  // transparent green
+  marker.color.b = 0.0;
+  marker.pose.orientation.w = 1.0;
+
+  auto polar_to_xyz = [](auto r, auto az, auto el) {
```

Comment:
super nit: variable names for az, el are not commonly used. Suggest using radius, azimuth, elevation.

Reply:
Renamed in [52ff3e8](https://github.com/autowarefoundation/autoware_universe/pull/11549/commits/52ff3e816c6c8cb1fd0a30a1dc420cf6d6ab4595).  Although other lines still have variables like `az1`, I left them as they are because they are short-scoped local variables, and avoiding the codes getting longer unintentionally. :pray: